packages/main/src/MultiComboBox.js

```diff
@@ -162,6 +164,19 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Defines the filter type of the <code>ui5-multi-combobox</code>.
+		 * Available options are: <code>StartsWithPerTerm</code>, <code>None</code>.
+		 *
+		 * @type {string}
+		 * @defaultvalue "StartsWithPerTerm"
+		 * @public
+		 */
+		filter: {
```

Comment:
This should not be string, but an enum

Reply:
MultiComboBox has not really a use case for e.g. contains filter, for the end user the API works fine (the same way as an enum), we can discuss this internally if it is worth to be changed.